common/cmake/devices.cmake

```diff
@@ -1571,105 +1762,18 @@ function(ADD_FPGA_TARGET)
 
   get_target_property_required(NO_BITSTREAM ${ARCH} NO_BITSTREAM)
   if(NOT ${NO_BITSTREAM})
-    # Generate bitstream
-    # -------------------------------------------------------------------------
-    get_target_property_required(BITSTREAM_EXTENSION ${ARCH} BITSTREAM_EXTENSION)
-    set(OUT_BITSTREAM ${OUT_LOCAL}/${TOP}.${BITSTREAM_EXTENSION})
-
-    if(${USE_FASM})
-      get_target_property_required(FASM_TO_BIT ${ARCH} FASM_TO_BIT)
-      get_target_property_required(FASM_TO_BIT_CMD ${ARCH} FASM_TO_BIT_CMD)
-      get_target_property(FASM_TO_BIT_DEPS ${ARCH} FASM_TO_BIT_DEPS)
-      if ("${FASM_TO_BIT_DEPS}" STREQUAL "FASM_TO_BIT_DEPS-NOTFOUND")
-        set(FASM_TO_BIT_DEPS "")
-      endif()
-      get_target_property(FASM_TO_BIT_EXTRA_ARGS ${BOARD} FASM_TO_BIT_EXTRA_ARGS)
-      if ("${FASM_TO_BIT_EXTRA_ARGS}" STREQUAL "FASM_TO_BIT_EXTRA_ARGS-NOTFOUND")
-        set(FASM_TO_BIT_EXTRA_ARGS "")
-      endif()
-
-      get_target_property_required(PYTHON3 env PYTHON3)
-
-      set(BITSTREAM_DEPS ${OUT_FASM} ${FASM_TO_BIT} ${FASM_TO_BIT_DEPS})
 
-      string(CONFIGURE ${FASM_TO_BIT_CMD} FASM_TO_BIT_CMD_FOR_TARGET)
-      separate_arguments(
-        FASM_TO_BIT_CMD_FOR_TARGET_LIST UNIX_COMMAND ${FASM_TO_BIT_CMD_FOR_TARGET}
-      )
+    get_target_property_required(DEVICE_TYPE ${DEVICE} DEVICE_TYPE)
```

Comment:
Not sure why this is here?  It doesn't get used?

Reply:
Also old code that I'm removing.